src/main/java/uk/gov/companieshouse/model/validator/AppealReasonValidator.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.companieshouse.model.validator;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
We should be using the structured logging library. Is this being covered in a different task?

Reply:
Can be removed entirely now that the validator no longer logs.

Reply:
Done